app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -341,14 +341,27 @@ const ruleShape = PropTypes.shape({
   type: PropTypes.string.isRequired,
 });
 
+const ruleField = PropTypes.shape({
```

Comment:
Let's avoid duplicate here, please move it to the its own variable and use in components where needed.